src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -130,11 +40,19 @@ const ChatItemMyTextV2 = ({
   message = '',
   status = ChatStatus.PENDING,
   avatar,
-  chatType
+  chatType,
+  messageType,
+  data
 }: ChatItemMyTextProps) => {
-  const {onContextMenuPressed} = useMessageHook();
-  const messageRef = React.useRef<Text>(null);
+  const {setReplyPreview, onContextMenuPressed} = useMessageHook();
+
+  const swipeableRef = React.useRef<Swipeable | null>(null);
   const [isNewLine, setIsNewLine] = React.useState(true);
+  const bubblePosition = useSharedValue(0);
+
+  const isReply = messageType === 'reply';
+  const isReplyPrompt = messageType === 'reply_prompt';
+  const isShowUserInfo = !isContinuous || isReplyPrompt || isReply;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `data` prop is not destructured in the old hunk but it's used in the new hunk. Make sure that this prop is passed correctly to the component and has the expected structure.



<!-- This is an auto-generated comment by OSS CodeRabbit -->